maps_dashboards/public/utils/geo_formater.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const geoJSONTypes: string[] = [
+  'point',
+  'linestring',
+  'polygon',
+  'multipoint',
+  'multilinestring',
+  'multipolygon',
+  'geometrycollection',
+];
+
+export function isGeoJSON(value: { type: any; coordinates: any }) {
+  if (!value) return false;
+  if (!value.type || !value.coordinates) {
+    return false;
+  }
+  const geoJSONType = value.type;
+  if (geoJSONTypes.includes(geoJSONType.toLowerCase())) {
+    return true;
+  }
+  return false;
+}
+
+export function isLonLatObject(value: any) {
+  if (value?.lat && value?.lon) {
+    return true;
+  }
+  return false;
+}
+
+export function wktPointToGeoJSON(value: any) {
+  if (typeof value === 'string') {
+    const removedType = value.trim().replace('POINT', '');
+    const coordinatesExpression = RegExp('[(](?<longitude>.+) (?<latitude>.+)[)]');
```

Comment:
I don't think this will cover WKT point format comprehensively. It will even parse `POINT (1a 3b)`.

Reply:
Good point. I was thinking about adding lat/lon check after extraction. However, this will be called after geo_point data is retrieved from OpenSearch. I believe users won't be able to save POINT( 1a 3b) to OpenSearch.

Reply:
Will not be supporting WKT for now.